tests/drivers/gpio/gpio_get_direction/src/main.c

```diff
@@ -6,7 +6,7 @@
 
 #include "test_gpio_get_direction.h"
 
-static void test_disconnect(void)
+ZTEST(gpio_get_direction, test_disconnect)
```

Comment:
Missing: `ZTEST_SUITE(gpio_get_direction, ...)`

Reply:
Weird - i distinctly remember adding that. Maybe I forgot to push

Reply:
May have been lost because i reordered commits. Fixed!